components/Base/Chips/Group/index.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="chips-group">
+    <button
+      v-for="item in values"
+      :key="item.value"
```

Comment:
hello @doohanas , it this `item.value` guaranteed unique?

Reply:
Hello, kak @Ibwedagama, I am not sure, is guaranteed unique or not.  I just realize because I don't have an `id` property, is it better to give the value of the `key is an `index`?

Reply:
yes, you're right @doohanas , vue will warn us if we give an element a same key. if you sure that `item.value` values will always be unique than that's okay, but if not I think it's better to generate unique `key` based on loop index like this:

```js
 v-for="(item, index) in value"
 :key="index"
```
or better :

```js
 v-for="(item, index) in value"
 :key="`chip-button-${index}`"

Reply:
Thanks for your suggestion kak @Ibwedagama. I think that `item.value` can be unique kak. Because, I read the explanation about give an index as the key in this [article](https://michaelnthiessen.com/understanding-the-key-attribute). What dou you think kak?

Reply:
Cool! it's a valid point in my opinion, the `index` will cause an issue with dynamically added or removed items 👍🏼 

Just stick with your approach @doohanas 👍🏼 👍🏼

Reply:
Appreciate it kak @Ibwedagama , thank you  :fire: :fire: :fire: